app/src/main/java/com/dicoding/picodiploma/sejiwaproject/model/League.kt

```diff
@@ -5,8 +5,8 @@ import kotlinx.android.parcel.Parcelize
 
 @Parcelize
 data class League(
-    var name: String = "",
-    var description: String = "",
-    var location: String = "",
-    var photo: Int = 0
+    var name: String? = "",
+    var description: String? = "",
+    var location: String?= "",
+    var photo: Int? = 0
 ) : Parcelable
```

Comment:
You have set this as nullable properties. But default value still exist. Changes it to immutable by using `val`.